src/core/client/stream/tabs/configure/components/MessageBoxConfig.tsx

```diff
@@ -0,0 +1,151 @@
+import { Localized } from "fluent-react/compat";
+import React, { StatelessComponent, Suspense } from "react";
+import { Field } from "react-final-form";
+
+import { MarkdownEditor } from "talk-framework/components/loadables";
+import { parseBool } from "talk-framework/lib/form";
+import {
+  MessageBox,
+  MessageBoxContent,
+  MessageBoxIcon,
+} from "talk-stream/components/MessageBox";
+import {
+  HorizontalGutter,
+  Icon,
+  Spinner,
+  TileOption,
+  TileSelector,
+  Typography,
+  ValidationMessage,
+} from "talk-ui/components";
+
+import ToggleConfig from "./ToggleConfig";
+import WidthLimitedDescription from "./WidthLimitedDescription";
+
+import styles from "./MessageBoxConfig.css";
+
+interface Props {
+  disabled: boolean;
+}
+
+const MessageBoxConfig: StatelessComponent<Props> = ({ disabled }) => (
+  <Field name="messageBox.enabled" type="checkbox" parse={parseBool}>
+    {({ input }) => (
+      <ToggleConfig
+        id={input.name}
+        name={input.name}
+        onChange={input.onChange}
+        onFocus={input.onFocus}
+        onBlur={input.onBlur}
+        checked={input.checked}
+        disabled={disabled}
+        title={
+          <Localized id="configure-messageBox-title">
+            <span>Enable Message Box for this Stream</span>
+          </Localized>
+        }
+      >
+        <HorizontalGutter size="oneAndAHalf">
+          <Localized id="configure-messageBox-description">
+            <WidthLimitedDescription>
+              Add a message to the top of the comment box for your readers. Use
+              this to pose a topic, ask a question or make announcements
+              relating to this story.
+            </WidthLimitedDescription>
+          </Localized>
+          {input.checked && (
+            <Field name="messageBox.icon">
+              {({ input: iconInput }) => (
+                <Field name="messageBox.content">
+                  {({ input: contentInput, meta }) => (
+                    <>
+                      <HorizontalGutter size="half" container="section">
+                        <Localized id="configure-messageBox-preview">
+                          <Typography
+                            variant="bodyCopyBold"
+                            container="h1"
+                            className={styles.preview}
+                          >
+                            PREVIEW
+                          </Typography>
+                        </Localized>
+                        <MessageBox>
+                          {iconInput.value && (
+                            <MessageBoxIcon>{iconInput.value}</MessageBoxIcon>
+                          )}
+                          <MessageBoxContent>
+                            {contentInput.value}
```

Comment:
We could probably just do this to keep the height of the box:
```suggestion
                            {contentInput.value || " "}
```